script.module.tvdbsimple/.github/workflows/addon-check.yml

```diff
@@ -0,0 +1,16 @@
+name: Kodi Addon-Check
```

Comment:
Please add `.github` to gitattributes so that it would be send over with your PRs

Reply:
Removed the folder